Lib/test/test_patma.py

```diff
@@ -4,6 +4,7 @@
 import enum
 import inspect
 import unittest
+import sys
```

Comment:
Minor: can you sort these?

```suggestion
import sys
import unittest
```